client/my-sites/subscribers/constants.ts

```diff
@@ -11,8 +13,17 @@ export enum SubscribersFilterBy {
 	WPCOM = 'wpcom',
 	Free = 'free',
 	Paid = 'paid',
-	EmailSubscriber = 'email_subscriber',
 	ReaderSubscriber = 'reader_subscriber',
+	UnconfirmedSubscriber = 'unconfirmed_subscriber',
+	EmailSubscriber = 'email_subscriber',
+	BlockedSubscriber = 'blocked_subscriber',
 }
 
+export const SubscribersStatus = {
+	Subscribed: translate( 'Subscribed' ),
```

Comment:
On the backend, the value is returned from SQL queryy and we need to filter on these values as well, so it's better to do the translation on Calypso.